ui/src/app/pages/analysis/notebook-redirect/component.ts

```diff
@@ -241,11 +242,11 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
     if (nbName.endsWith('.ipynb')) {
```

Comment:
I think this check can be removed now, since drop/append are idempotent.  Starting from 240, we can do something like:
```
const decodedName = decodeURIComponent(nbName);
this.notebookName = dropNotebookFileSuffix(decodedName);
this.fullNotebookName = appendNotebookFileSuffix(decodedName);
```

Reply:
oo. good catch!